robot_sort/robot_sort.py

```diff
@@ -103,34 +103,62 @@ def sort(self):
         keep moving till you find an item that is greater than and then swap
         continue moving through the list until it is sorted
         """
-       
-        self.set_light_on()
-
+        self.set_light_on()  # First, set the lights on
```

Comment:
Great job Frulwinn. I think we went over all the code in our one on one, so I'll merge this in. In the future, we'll find that mystery bug sometime!